make/include/defaults

```diff
@@ -13,7 +13,7 @@ export CF_NAMESPACE="${NAMESPACE}"
 : "${DOMAIN:=cf-dev.io}"
 : "${UAA_NAMESPACE:=uaa}"
 : "${UAA_HOST:=uaa.${DOMAIN}}"
-: "${UAA_PORT:=2793}"
+export UAA_PORT="2793"
```

Comment:
This is no longer customizable? No chance of somebody running against a non-default UAA?

Reply:
Using our very Vagrant-tied make scripts? No.